core/templates/dev/head/components/ExplorationSummaryTileDirective.js

```diff
@@ -78,6 +79,13 @@ oppia.directive('explorationSummaryTile', [function() {
           }
         );
 
+        if ($scope.isCommunityOwned) {
+          $scope.contributors.unshift('Community Owned');
```

Comment:
While this does work, the modelling is wrong: "Community Owned" is not a contributor. Instead, I suggest that you create another $scope variable (avatarList, for example) and populate it accordingly, then refer to that in the HTML. In general, it's very important, for clarity of the codebase, that the variable name accurately reflects what's in it, otherwise this is going to cause issues for future developers.